data_steward/deid/config/ids/tables/drug_exposure.json

```diff
@@ -1,8 +1,20 @@
pt_code like 'Y38%' OR concept_code like 'X52%' OR concept_code like 'V%' OR concept_code like 'W65%' OR concept_code like 'W66%' OR concept_code like 'W67%' OR concept_code like 'W68%' OR concept_code like 'W69%' OR concept_code like 'W70%' OR concept_code like 'W71%' OR concept_code like 'W72%' OR concept_code like 'W73%' OR concept_code like 'W74%' OR concept_code like 'X92%' OR concept_code like 'X93%' OR concept_code like 'X94%' OR concept_code like 'X95%' OR concept_code like 'X96%' OR concept_code like 'X97%' OR concept_code like 'X98%' OR concept_code like 'X99%' OR concept_code like 'Y00%' OR concept_code like 'Y01%' OR concept_code like 'Y02%' OR concept_code like 'Y03%' OR concept_code like 'Y04%' OR concept_code like 'Y05%' OR concept_code like 'Y06%' OR concept_code like 'Y07%' OR concept_code like 'Y08%' OR concept_code like 'Y09%')) union all select concept_id_2 from `:idataset.concept` join `:idataset.concept_relationship` rel on concept_id = concept_id_1 where (vocabulary_id='ICD9CM' and (concept_code like 'E80%' OR concept_code like 'E81%' OR concept_code like 'E82%' OR concept_code like 'E83%' OR concept_code like 'E84%' OR concept_code like 'E913%' OR concept_code like 'E910%' OR concept_code like 'E928.0' OR concept_code like 'E95%' OR concept_code like 'E96%' OR concept_code like 'E97%' OR concept_code like 'E99%' OR concept_code = '799.9' OR concept_code = '799' OR concept_code = '798' OR concept_code like 'V3%' OR concept_code like '764%' OR concept_code like '765%' OR concept_code like '766%' OR concept_code like '767%' OR concept_code like '768%' OR concept_code like '769%' OR concept_code like '770%' OR concept_code like '771%' OR concept_code like '772%' OR concept_code like '773%' OR concept_code like '774%' OR concept_code like '775%' OR concept_code like '776%' OR concept_code like '777%' OR concept_code like '778%' OR concept_code like '779%')) OR (vocabulary_id='ICD10CM' and (concept_code like 'P%' OR concept_code like 'Z38%' OR concept_code = 'R99' OR concept_code like 'Y36%' OR concept_code like 'Y36%' OR concept_code like 'Y37%' OR concept_code like 'Y35%' OR concept_code like 'Y38%' OR concept_code like 'X52%' OR concept_code like 'V%' OR concept_code like 'W65%' OR concept_code like 'W66%' OR concept_code like 'W67%' OR concept_code like 'W68%' OR concept_code like 'W69%' OR concept_code like 'W70%' OR concept_code like 'W71%' OR concept_code like 'W72%' OR concept_code like 'W73%' OR concept_code like 'W74%' OR concept_code like 'X92%' OR concept_code like 'X93%' OR concept_code like 'X94%' OR concept_code like 'X95%' OR concept_code like 'X96%' OR concept_code like 'X97%' OR concept_code like 'X98%' OR concept_code like 'X99%' OR concept_code like 'Y00%' OR concept_code like 'Y01%' OR concept_code like 'Y02%' OR concept_code like 'Y03%' OR concept_code like 'Y04%' OR concept_code like 'Y05%' OR concept_code like 'Y06%' OR concept_code like 'Y07%' OR concept_code like 'Y08%' OR concept_code like 'Y09%')))) "
```

Comment:
Same thing here-- not finding the alias `drug_exposure` in `drug_exposure.drug_exposure_id = record2.drug_exposure_id`.